src/main/redux/sagas/api/publication/import/importLcplFromFs.ts

```diff
@@ -61,7 +61,7 @@ export function* importLcplFromFS(
             try {
                 if (r2LCP.Rights.Start) {
                     if (moment(r2LCP.Rights.Start).isAfter(now)) {
-                        res = 11;
+                        res = 1111;
```

Comment:
That's fine, but please add a code comment:
```
// Normally: LICENSE_OUT_OF_DATE 11
// Note that this is reserved already: LICENSE_SIGNATURE_DATE_INVALID 111
```

Reply:
I wonder whether we need similar logic here:
https://github.com/edrlab/thorium-reader/blob/cf04170d65959b66cef906f3cbe362d8dbbd73c0/src/main/redux/sagas/publication/openPublication.ts#L102-L104
and:
https://github.com/edrlab/thorium-reader/blob/cf04170d65959b66cef906f3cbe362d8dbbd73c0/src/main/redux/sagas/publication/openPublication.ts#L188-L190
and:
https://github.com/edrlab/thorium-reader/blob/cf04170d65959b66cef906f3cbe362d8dbbd73c0/src/main/api/lcp.ts#L66-L68